controllers/datasciencecluster/datasciencecluster_controller.go

```diff
@@ -308,22 +329,16 @@ func (r *DataScienceClusterReconciler) reconcileDashboardComponent(ctx context.C
 		}
 	}
 
-	// Create the Dashboard instance
-	dashboard := dashboardctrl.GetDashboard(instance)
-
-	// Reconcile component
-	err := r.apply(ctx, instance, dashboard)
-
 	if err != nil {
-		r.Log.Error(err, "Failed to reconcile Dashboard component")
+		r.Log.Error(err, "Failed to reconcile component: "+componentName)
 		instance = r.reportError(err, instance, fmt.Sprintf("failed to reconcile %s on DataScienceCluster", componentName))
 		instance, _ = status.UpdateWithRetry(ctx, r.Client, instance, func(saved *dscv1.DataScienceCluster) {
 			status.SetComponentCondition(&saved.Status.Conditions, componentName, status.ReconcileFailed, fmt.Sprintf("Component reconciliation failed: %v", err), corev1.ConditionFalse)
 		})
 		return instance, err
 	}
 
-	r.Log.Info("Dashboard component reconciled successfully")
+	r.Log.Info("component reconciled successfully: " + componentName)
```

Comment:
not for this PR, but we can probably squeeze the 3 `UpdateWithRetry` calls in a single one